ui/darksky/README.md

```diff
@@ -6,11 +6,17 @@ the card's capabilities through configuration flags.
 
 **Installation**
 ------------------------------
-1. Add **dark-sky-weather-card.js** to your **www/custom_ui/** directory.  If you don't have this directory (this is your first
-custom card), you will need to create it.
+1. Add ```dark-sky-weather-card.js``` to your ```<config-dir>*www/custom_ui/``` directory.  If you don't have this directory (this is your first custom card), you will need to create it.
 
-2. Download the amcharts icons from https://www.amcharts.com/dl/svg-weather-icons/ and put them in **www/icons/weather_icons**.  Create 
-the directories if necessary. You should end up with an **animated** directory and a **static** directory inside the **weather_icons** directory.
+2. Download the amcharts icons from https://www.amcharts.com/dl/svg-weather-icons/ and put them in ```<config-dir>www/icons/weather_icons```.  Create the directories if necessary.
+
+You should end up with the following folders:
+
+```<config-dir>/www/custom/ui/dark-sky-weather-card.js```
```

Comment:
~~~~
You should end up with the following folders:

 ```<config-dir>/www/custom/ui/dark-sky-weather-card.js```
~~~~
 Not a directory ... and the path is incorrect   .../custom_ui/   not .../custom/ui/